### PR TITLE
adding regression test for FV3_RRFS_v0

### PR DESCRIPTION
**Pull request by panll**
adding regression test for FV3_GSD_SAR_v1
added files:
./tests/tests/fv3_ccpp_gsd_sar_v1
./tests/fv3_conf/ccpp_gsd_sar_v1_run.IN

modified file:
tests/rt_ccpp_gsd.conf

### CODE REVIEW

**Comment by panll** [comment]
adding one test for suite RRFS_v0 for SRWeather release
modified file:
         tests/rt_ccpp_gsd.conf
added files:
        tests/tests/fv3_ccpp_rrfs_v0
        tests/fv3_conf/ccpp_rrfs_v0_run.IN
        parm/ccpp_rrfs_v0.nml.IN

Tested on hera.

**Comment by climbfuji** [comment]
@panll do you want to update this PR as discussed last week?

**Comment by panll** [comment]
yes, I'll submit the update soon. Thanks! @climbfuji

File: tests/tests/fv3_ccpp_gsd_sar_v1 [hunk 1]
```diff
@@ -0,0 +1,37 @@
+###############################################################################
```
Comment (climbfuji) [inline tests/tests/fv3_ccpp_gsd_sar_v1:1]: If we are adding a new test fv3_ccpp_gsd_sar_v1, then we will also want to run it as part of rt_ccpp_gsd.conf, or?

**Comment by panll** [comment]
Yes, it needs to turn on rt_ccpp_gsd.conf. hera is down today. I can turn it on after hera comes back. Thanks! @climbfuji

**Comment by panll** [comment]
GSD_SAR_v1 test is also included and ready to merge. @climbfuji

**Comment by climbfuji** [comment]
Associated PRs:

https://github.com/NCAR/fv3atm/pull/45
https://github.com/NCAR/ufs-weather-model/pull/43
https://github.com/NCAR/ufs-weather-model/pull/49

For regression testing information, see https://github.com/NCAR/ufs-weather-model/pull/49.

**Pull request merged**